bono.yaml

```diff
@@ -36,18 +36,38 @@ description: >
   Clearwater Bono node
 
 parameters:
-  public_net_id:
+  public_mgmt_net_id:
     type: string
-    description: ID of public network
+    description: ID of public management network
     constraints:
       - custom_constraint: neutron.network
         description: Must be a valid network ID
-  private_net_id:
+  private_mgmt_net_id:
     type: string
-    description: ID of private network
+    description: ID of private management network
     constraints:
       - custom_constraint: neutron.network
         description: Must be a valid network ID
+  public_sig_net_id:
+    type: string
+    description: ID of public signaling network
+    constraints:
+      - custom_constraint: neutron.network
+        description: Must be a valid network ID
+  private_sig_net_id:
+    type: string
+    description: ID of private signaling network
+    constraints:
+      - custom_constraint: neutron.network
+        description: Must be a valid network ID
+  private_sig_net_cidr:
```

Comment:
Correct.  I've added a comment, but where we use the variable, rather than here - I think that's implementation detail.